v3/go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/newrelic/go-agent/v3
 go 1.7
 
 require (
-	github.com/golang/protobuf v1.3.3
-	google.golang.org/grpc v1.27.0
+	github.com/golang/protobuf v1.4.3
+	google.golang.org/grpc v1.39.0
```

Comment:
If we're requiring new versions of these libraries, will this affect our customers in any way?

Reply:
I think we can move the version back. That was automatically bumped by my go runtime, not because I purposefully upgraded it.